### DISCUSSION TITLE
Unable to setup discussions in some repos

### DISCUSSION THREAD

**Discussion by nicwest** [community/community#3519] [Discussions]
I'm getting this error when attempting to enable discussions in a private repo.

![2021-03-10-093959_1153x473_scrot](https://user-images.githubusercontent.com/794070/110609084-c5da0d00-8184-11eb-83ab-09f01cdd504a.png)

Am I doing something stupid?

Notably there is no button that allows starting a new discussion.

**Comment by AlBannaTechno** [comment]
Try to create an issue, then convert it to a discussion, this trick may work.

**Reply by nicwest** [reply]
you can convert issues that appears to work, but still no button to start new discussions directly:

![2021-03-10-131401_1261x208_scrot](https://user-images.githubusercontent.com/794070/110634997-88847800-81a2-11eb-8561-765592dc1031.png)
